crates/language/src/language.rs

```diff
@@ -246,10 +246,15 @@ pub trait Language: NodeTypes {
         &EXACT_REPLACED_VARIABLE_REGEX
     }
 
-    fn is_comment(&self, _id: SortId) -> bool;
-
-    fn is_comment_wrapper(&self, _node: &tree_sitter::Node) -> bool {
-        false
+    // checks if the sort is the languages comment sort
+    fn is_comment_sort(&self, _id: SortId) -> bool;
+
+    // checks if a node is a comment. distinct from the above
+    // in that sometimes a node is a comment but doesn't have
+    // a comment sort for example when parsing javascript,
+    // comments embedded in jsx dont have the comment sort
```

Comment:
I think we realistically want to get rid of `is_comment_sort`, since all uses of it we'd also want to be finding JSX comment nodes too.

Reply:
No when compiling we don't care if the node is a comment we just want the sort.